### ISSUE TITLE
Make the left hand side menu smaller in width

### ISSUE THREAD

**Issue by davsclaus** [fabric8io/fabric8-console#114]
It take up too much space by default, so tables etc get crambled together a bit on screens. There is a fair bit of whitespace and the menu names are small. So maybe reduce the size to 50 or 60% as of now.

![screen shot 2016-03-01 at 7 23 59 pm](https://cloud.githubusercontent.com/assets/477100/13437212/5415051e-dfe3-11e5-8888-c51e0fb52e4c.png)

There is no slider so you cannot resize it yourself.

It helps for demos where you run on lower resolution even so, to make the left menu not take up so much white space.

**Comment by gashcrumb** [comment]
Yeah, really we've gotta update to a newer patternfly which has a better side-menu widget.  I've been procrastinating a bit since that update also renames a bunch of their directives, some of which we use :-(

**Comment by jimmidyson** [comment]
Would be cool to switch to a vertical, icon only left menu on narrow displays. Think patternfly recommends that or similar too.

Or even an off canvas approach for narrow displays with nav toggle button? With nav docked on wide displays.

There are a lot of problems running the console on narrow displays in general, overlapping elements, etc but this menu one. I'd prefer to stick to a fixed width rather than fluid, with width set via media queries (sure there's a short name for that technique!). That way it's way easier to test layouts properly & ensure stuff works on all display types.

**Comment by davsclaus** [comment]
Also make some of the main screens like pod less "noisy" - we have too many labels in there so the pods are spaced far apart because there is like 5+ labels on the pods. 

For the number of labels its a matter of adjusting all the quickstarts to what naming concention we are using. It used to be less until the CD stuff added for version and artifect etc (the blue ones) so we have duplicates a lot.

**Comment by davsclaus** [comment]
And who like that navy blue background color in the filter box - must be one of the worst color choices.

**Comment by gashcrumb** [comment]
Yeah, lemme see about switching that side-panel over to the patternfly one for starters the one that's there is just a placeholder really.

**Comment by gashcrumb** [comment]
Hmm, well here's a start on using the new sidebar stuff.  It's a bit smaller at least.  A few issues to iron out though.
![capture](https://cloud.githubusercontent.com/assets/351660/13443515/5566e40a-dfcf-11e5-9000-3ff1f1438b5d.PNG)

**Comment by jimmidyson** [comment]
Looking great!

**Comment by gashcrumb** [comment]
K, just pushed updates across a few projects, have it working nicely...  We should figure out icons for all the menu items so the menu collapses nicely, looks a bit odd in 'Runtime' for example since there's no icons.  Also I resorted to using icons in the top nav for the moment for 'Projects' and 'Runtime' for the moment.  Anyways, if you pull in the 'latest' tag it should look kinda like:
![capture](https://cloud.githubusercontent.com/assets/351660/13467746/8e8b5fd8-e06d-11e5-9a1a-edc58a3b928f.PNG)
